command/meta.go

```diff
@@ -216,6 +216,12 @@ func (m *ApiMeta) InitializeClient() (*http.Client, error) {
 			// Misconfigured.
 			return nil, errors.New("Incorrect x509 auth configuration.\nMust specify certPath/keyPath or cert/key pair.")
 		}
+	} else if auth != nil && auth.Enabled && auth.Basic != nil {
```

Comment:
Should use the `isValid()` for basic auth here, currently unused.

Reply:
👍 fix incoming!